config/targets.js

```diff
@@ -1,3 +1,18 @@
+'use strict';
+
+const browsers = [
+	'last 1 Chrome versions',
+	'last 1 Firefox versions',
+	'last 1 Safari versions'
```

Comment:
Since chrome and firefox have auto-update feature it is safe to use only one version, but with Safari I'd check our compatibility table:

http://community.wikia.com/wiki/Help:Supported_browsers

iOS9 is the last supported so let's make sure Safari for this version of iOS is also supported here.